nuget/helpers/lib/NuGetUpdater/Directory.Common.props

```diff
@@ -9,7 +9,7 @@
             NuGetUpdater\NuGetUpdater.Core\FrameworkChecker\SupportedFrameworks.cs
       2. Update tests as needed at `NuGetUpdater\NuGetUpdater.Core.Test\FrameworkChecker\CompatibilityCheckerFacts.cs`
     -->
-    <CommonTargetFramework>net8.0</CommonTargetFramework>
+    <CommonTargetFramework>net9.0</CommonTargetFramework>
```

Comment:
Despite the comments above, the referenced files haven't been updated to explicitly add `net9.0`, and the changes for .NET 8 were only made 6 months ago, way after it was released: https://github.com/NuGet/NuGetGallery/pull/9858